### PR TITLE
To-Do list Milestone 3: Create interactive list

### PR DESCRIPTION
**Pull request by andres-condezo**
## To-Do list Milestone 3: Create interactive list

In this milestone we:

- Updated styles.css file eith mobile and desktop media query.
- Created the check box listeners function and the logic for the "delete completed task"  function.

### CODE REVIEW

File: src/modules/app.js [hunk 1]
```diff
@@ -78,20 +98,52 @@ class ToDoApp {
...
+        this.completedTaskArr.push(index);
+      } else {
+        this.completedTaskArr = this.completedTaskArr.filter((el) => el !== index);
+      }
+      $check.classList.toggle('hidden');
+      $square.classList.toggle('hidden');
+      $textTask.classList.toggle('underlined');
+      this.taskArr[index].completed = !this.taskArr[index].completed;
+      this.saveLocalStorage();
+      this.checkCompletedArr();
```
Comment (ZeenatLawal) [inline src/modules/app.js]: - [ ] The status of the task should be updated when the checkbox is clicked and the status should also be stored in the local storage. Currently, the status changes back to false when refreshed, kindly modify that.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @andres-condezo, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/app.js (hunk 1 above)
Comment (andres-condezo) [reply src/modules/app.js:117]: ## Hi @ZeenatLawal 👋,
Thanks for finding the bug, I'll fix it.

Comment (ZeenatLawal) [reply src/modules/app.js]: @andres-condezo You are welcome. Best of luck 👍🏼

**Comment by andres-condezo** [comment]
## Hi @ZeenatLawal  👋;
Thanks for finding the bug, I'll fix it.

File: src/modules/app.js (hunk 1 above)
Comment (DeliceLydia) [reply src/modules/app.js]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @andres-condezo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**